client/src/components/VideoPlayer.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class VideoPlayer extends React.Component {
```

Comment:
Every component is a class from what I am seeing. Maybe refactor depending on if the component needs state? Stateless components serve their purpose as well.

Reply:
Great point; I copy+pasted all these components in quick succession without much thought as to functional vs. class style. I'll reassess the needs of each component.